examples/Dockerfile.service

```diff
@@ -2,13 +2,7 @@
 #   You will need to be logged into the registry in order to build this.
 ARG REGISTRY=562955126301.dkr.ecr.eu-west-2.amazonaws.com
 ARG BASE_VERSION=1.0.0
-ARG DEV_VERSION=1.0.0
-FROM $REGISTRY/ffc-node-base:$BASE_VERSION
-
-# Make sure that we don't run as root. The "node" user is set up in the base node version that our base parent image uses.
-USER node
-
-WORKDIR /home/node
+FROM $REGISTRY/ffc-node-base:$BASE_VERSION as base

```

Comment:
Nitpicking: `as` should be capitalised because it's part of the Dockerfile syntax:
```
FROM $REGISTRY/ffc-node-base:$BASE_VERSION as base
```